### ISSUE TITLE
[Components & Elements]: Should we use a maturity scale?

### ISSUE THREAD

**Issue by joshbruce** [GSA/sam-web-design-standards#20]
## Issue type

* All components and elements

## What happened

For release 0.0.10 of the US Web Design Standards, a maturity model was put in place for the UI components: [https://standards.usa.gov/about-our-work/component-maturity-scale/](https://standards.usa.gov/about-our-work/component-maturity-scale/)

Should we follow suit? Or, would this be overkill at this juncture given that these Standards mainly apply to those working on the SAM environment?

**Comment by jagannathch** [comment]
I think this is a good idea to implement so that designers/non-designers are on the same page as to where we are with the design patterns.

**Comment by joshbruce** [comment]
@jagannathch - Might be worthwhile to mark whether the element or component is already in the kit or not...? Thinking to the right of the maturity marker.

<img width="747" alt="screen shot 2016-08-13 at 9 29 00 pm" src="https://cloud.githubusercontent.com/assets/15252830/17646563/0616b362-619d-11e6-80e3-77265d9c978a.png">

This way we can easily:

1. Communicate to internal and external stakeholders how much we think this element could change in the future.
2. Communicate to those leveraging the kit on whether or not it has been defined there.

**Comment by jagannathch** [comment]
@joshbruce good idea...I think between the maturity market and an availability label, we can definitely imply both your points...

**Comment by diego-ruiz-rei** [comment]
definitely think this is worthwhile to implement
